src/substrate/filters/type_parser.ts

```diff
@@ -13,6 +13,12 @@ export function ParseType (
   // TODO: we can unify this with the enricher file: parse out the kind, and then
   //   marshall the rest of the types in the same place. But for now, we can leave as-is.
   switch (section) {
+    case 'imonline':
```

Comment:
Did you test this to confirm that it's `imonline` and not `im-online`?

Reply:
Ah, typo.

Reply:
I usually see the module name from [here](https://github.com/paritytech/substrate/tree/00a400f82539e2f78e8ddbcd98aea512c87c5f3c/frame) is there a better place?

Reply:
The module name gets changed sometimes in the API, so the best way is to set up an event listener and log all events, then take the name from there.